gapic-generator/lib/google/gapic/schema/wrappers.rb

```diff
@@ -217,11 +217,9 @@ def host
           options[:".google.api.default_host"] if options
         end
 
-        # @return [Google::Api::OAuth] OAuth information for the client.
-        #   The "scopes" key is a repeated string; see
-        #   `google/api/metadata.proto`.
-        def oauth
-          options[:".google.api.default_host"] if options
+        # @return [String] The OAuth scopes information for the client.
+        def scopes
+          options[:".google.api.oauth_scopes"] if options
```

Comment:
This is a comma separated string. Maybe it would be best to do the split here and return an array?

Reply:
It is a single string with commas. Updated to return an array of scope strings.